drivers/i2c/i2c_ll_stm32.h

```diff
@@ -105,7 +105,8 @@ struct i2c_stm32_data {
 	const struct device *smbalert_cb_dev;
 #endif
 #ifdef CONFIG_I2C_STM32_V2_DMA
-	struct dma_config dma_cfg;
+	struct dma_config dma_tx_cfg;
+	struct dma_config dma_rx_cfg;
```

Comment:
Nit (non-blocking): since struct `i2c_stm32_config` has `tx_dma` and `rx_dma`, it would be more logical to name these `tx_dma_cfg` and `rx_dma_cfg` (or the other way around by renaming the first two as `dma_tx` and `dma_rx`)